openid-caep-interoperability-profile-1_0.md

```diff
@@ -225,9 +218,9 @@ All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bi
 ### OAuth Scopes
 Depending on the features supported by the OAuth service and the SSF APIs, the client SHALL discover the OAuth scopes as follows:
 
-1. If the Resource Server, hosting SSF configuration APIs, supports OAuth Protected Resource Metadata {{OPRM}} then the client MUST obtain the required scopes by using it.
+1. If the Resource Server, hosting SSF configuration APIs, supports OAuth Protected Resource Metadata {{RFC8414}} then the client MUST obtain the required scopes by using it.
```

Comment:
Actually, this doesn't look like the right reference. The OPRM document hasn't yet been assigned an RFC number, although it is in the RFC Ed queue: https://datatracker.ietf.org/doc/draft-ietf-oauth-resource-metadata/

Reply:
whoops. Good catch, thanks Atul! I'm sure someone told me it was published now! I obviously wasn't paying close enough attention when I googled for the  RFC number.

Have reverted those parts of my changes - should be good to go now hopefully!